pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```
@@ -1461,6 +1461,20 @@ spec:
                         pattern: ^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\/([0-9]|[1-2][0-9]|3[0-2])$|^s*((([0-9A-Fa-f]{1,4}:){7}(:|([0-9A-Fa-f]{1,4})))|(([0-9A-Fa-f]{1,4}:){6}:([0-9A-Fa-f]{1,4})?)|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{1,4}){0,1}):([0-9A-Fa-f]{1,4})?))|(([0-9A-Fa-f]{1,4}:){4}(((:[0-9A-Fa-f]{1,4}){0,2}):([0-9A-Fa-f]{1,4})?))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){0,3}):([0-9A-Fa-f]{1,4})?))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,4}){0,4}):([0-9A-Fa-f]{1,4})?))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){0,5}):([0-9A-Fa-f]{1,4})?))|(:(:|((:[0-9A-Fa-f]{1,4}){1,7}))))(%.+)?s*/([0-9]|[1-9][0-9]|1[0-1][0-9]|12[0-8])$
                         type: string
                       type: array
+                    fromCIDRGroupRefs:
+                      description: FromCIDRGroupRef is a list of CIDRs which the endpoint
+                        subject to the rule is allowed to receive connections from
+                        in addition to FromEndpoints, along with a list of subnets
```

I'm not sure what is meant by "along with a list of subnets contained within their corresponding IP block from which traffic should not be allowed". Isn't cidrgroup just a list of CIDRs? Does it have a more complicated structure that I'm not understanding? If my understanding is correct then I think this sentance is unnecessary.

```suggestion
                        (in addition to FromEndpoints).
```

---

No, you're right it's just a copy and paste error from `FromCIDRSet` doc. I'll fix it.